charts/safe-transaction-service/templates/deployment-contracts.yaml

```diff
@@ -42,18 +42,24 @@ spec:
           capabilities:
             drop:
             - ALL
-        {{- if .Values.resources }}
-        resources: {{- toYaml .Values.resources | nindent 12 }}
+        {{- if .Values.resources.contracts }}
+        resources: {{- toYaml .Values.resources.contracts | nindent 12 }}
         {{- else }}
         resources:
           limits:
-            cpu: 2
-            memory: 4G
+            cpu: 4
+            memory: 16G
           requests:
             cpu: 1
             memory: 2G
```

Comment:
It'd be better to have this configurable through the values.yaml.

Reply:
It is, see a few lines on top. I just wanted to add better defaults.